eve/tests/methods/get.py

```diff
@@ -899,6 +899,56 @@ def test_get_allowed_filters_operators(self):
         response, status = self.get(self.known_resource, where)
         self.assert400(status)
 
+    def test_get_nested_filter_operators_unvalidated(self):
+        """ test that nested filter operators are working correctly.
+        """
+        where = ''.join(('?where={"$and":[{"$or":[{"fldA":"valA"},',
```

Comment:
I agree! I also don't like it at all.

Any suggestions how this string:

    ?where={"$and":[{"$or":[{"fldA":"valA"},{"fldB":"valB"}]},{"fld2":"val2"}]}

or this (even prettier):

    ?where={
      "$and": [
        {
          "$or": [
            {
              "fldA": "valA"
            },
            {
              "fldB": "valB"
            }
          ]
        },
        {
          "fld2": "val2"
        }
      ]
    }

can be typed as a python statement that passes flake8 line length conventions and doesn't use a ton of concatenations?

I tried multiple combinations, that none were prettier and decided to move along into more interesting things than keep playing with flake8... :)